### ISSUE TITLE
U+03C6 and U+03D5 swapped (Forms of the Greek letter Phi)

### ISSUE THREAD

**Issue by gandaro** [adobe-fonts/source-code-pro#120]
According to the Unicode Standard, U+03C6 should use the loopy form of the Greek letter Phi and U+03D5 should use the straight form of the letter. Source Code Pro uses the straight form for U+03C6 and the loopy form for U+03D5.

Excerpt from the relevant paragraphs in the Unicode Standard:

> For mathematical and technical use, the straight form of the small phi is an important symbol  and  needs  to  be  consistently  distinguishable  from  the  loopy  form. […]
>
> […] fonts that also  intend  to  support  technical  use  of  the  Greek  letters  should  use  the  loopy  form [for U+03C6] to ensure appropriate contrast with the straight form used for U+03D5.

See [Unicode 8.0.0, Chapter 7.2, “Representative Glyphs for Greek Phi”](http://www.unicode.org/versions/Unicode8.0.0/ch07.pdf#G15531)

**Comment by miguelsousa** [comment]
This issue relates to [issue #70](https://github.com/adobe-fonts/source-sans-pro/issues/70) in the Source Sans repository.

The shape of U+03C6 can be the loop form or the full stroke form. The form used will depend on the genre of the design and on the designer's preferences.
On the other hand, the shape of U+03D5 should indeed be the straight form, no matter the style/genre of the typeface.

**Comment by gandaro** [comment]
Thanks for your effort. It seems like U+03C6 and U+03D5 look the same now, though. Was that intended?

**Comment by pauldhunt** [comment]
Their extender lengths are different to harmonize with their respective scripts. How would you expect them to differ?

**Comment by gandaro** [comment]
U+03C6 to use the curly form, as Unicode suggests:

> […] fonts that also intend to support technical use of the Greek letters should use the loopy form [for U+03C6] to ensure appropriate contrast with the straight form used for U+03D5.

See [Unicode 8.0.0, Chapter 7.2, “Representative Glyphs for Greek Phi”](http://www.unicode.org/versions/Unicode8.0.0/ch07.pdf#G15531)

**Comment by pauldhunt** [comment]
I thought I had fixed this. If it is still an error, then I apologize deeply. I will take another crack on it on the next update. Thanks for bringing this to my attention. 🙇

**Comment by moyogo** [comment]
No! U+03C6 is not the loopy form. U+03C6 can have whatever forms fits the style the type designer chooses, it just happens to be the loopy form in most fonts and therefore the reference glyph in the Unicode charts uses that. U+03D5 is a Math symbol and should have the straight form.

The full sentence in TUS 8.0 is
> Fonts used primarily for Greek text may use either glyph form for U+03C6, but fonts that
also intend to support technical use of the Greek letters should use the loopy form to
ensure appropriate contrast with the straight form used for U+03D5.

**Comment by gandaro** [comment]
Yes, it is optional, but Unicode recommends it for fonts that are intended to support technical use – as Source Code Pro certainly does. If you don’t like it though, you can leave it as it is now.
